### ISSUE TITLE
Does not works with Electron 5.0.5 ?

### ISSUE THREAD

**Issue by popod** [atom/node-keytar#195]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom>
    * Checked that there is not already an Atom package that provides the described functionality: <https://atom.io/packages>

### Description

Running and electron app on Windows 10 does this error:

```
\C:\Users\xxx\AppData\Local\Temp\4037a91f-06e5-4ddc-bff3-914e0958bcb9.tmp.node is not a valid Win32 application.
\\?\C:\Users\xxx\AppData\Local\Temp\4037a91f-06e5-4ddc-bff3-914e0958bcb9.tmp.node
    at process.func [as dlopen] (electron/js2c/asar.js:155)
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:850)
    at Object.func [as .node] (electron/js2c/asar.js:164)
    at Module.load (internal/modules/cjs/loader.js:677)
    at tryModuleLoad (internal/modules/cjs/loader.js:609)
    at Function.Module._load (internal/modules/cjs/loader.js:601)
    at Module.require (internal/modules/cjs/loader.js:715)
    at require (internal/modules/cjs/helpers.js:14)
    at Object.<anonymous> (C:\Users\xxx\AppData\Local\Programs\xxx\resources\app.asar\node_modules\keytar\lib\keytar.js:1)
    at Object.<anonymous> (C:\Users\xxx\AppData\Local\Programs\xxx\resources\app.asar\node_modules\keytar\lib\keytar.js:61)
```

Electron-builder: 20.44.4
Electron: 5.0.5
Node-keytar: 4.10.0
Node: 12.4.0

**Comment by Eiarzate1** [comment]
Having same exact issue on Windows 10.

My dependencies:

```
"electron": "^5.0.5",
"electron-builder": "^20.44.4",
"electron-rebuild": "^1.8.5",
"keytar": "^4.10.0"
```

**Comment by popod** [comment]
It seems that #196 describe the error reason.

@shiftkey have you time to check this ? Thanks

**Comment by shiftkey** [comment]
> @shiftkey do you have some time for checking this ? Thanks

I've not had time, but I can see [this Appveyor build](https://ci.appveyor.com/project/Atom/node-keytar/builds/25345644) only uploads the x86 prebuilds to the GitHub release:

```
prebuild info build Uploaded 13 new prebuild(s) to Github
prebuild info build -> prebuilds/keytar-v4.10.0-electron-v53-win32-ia32.tar.gz
prebuild info build -> prebuilds/keytar-v4.10.0-electron-v54-win32-ia32.tar.gz
prebuild info build -> prebuilds/keytar-v4.10.0-electron-v57-win32-ia32.tar.gz
prebuild info build -> prebuilds/keytar-v4.10.0-electron-v64-win32-ia32.tar.gz
prebuild info build -> prebuilds/keytar-v4.10.0-electron-v69-win32-ia32.tar.gz
prebuild info build -> prebuilds/keytar-v4.10.0-electron-v70-win32-ia32.tar.gz
prebuild info build -> prebuilds/keytar-v4.10.0-node-v48-win32-ia32.tar.gz
prebuild info build -> prebuilds/keytar-v4.10.0-node-v51-win32-ia32.tar.gz
prebuild info build -> prebuilds/keytar-v4.10.0-node-v57-win32-ia32.tar.gz
prebuild info build -> prebuilds/keytar-v4.10.0-node-v59-win32-ia32.tar.gz
prebuild info build -> prebuilds/keytar-v4.10.0-node-v64-win32-ia32.tar.gz
prebuild info build -> prebuilds/keytar-v4.10.0-node-v67-win32-ia32.tar.gz
prebuild info build -> prebuilds/keytar-v4.10.0-node-v72-win32-ia32.tar.gz
```

This was upgraded in https://github.com/atom/node-keytar/pull/187 so that's definitely a clue here.

**Comment by shiftkey** [comment]
@popod thanks for that clue about the prebuilds. This should be fixed with `v4.11.0` that is being published right now
